scopesim/__init__.py

```diff
@@ -63,4 +63,7 @@
 #                          VERSION INFORMATION                                #
 ###############################################################################
 
-__version__ = metadata.version(__package__)
+try:
+    __version__ = metadata.version(__package__)
+except:
+    version = 0.8
```

Comment:
```suggestion
    version = "undetermined"
```

We agreed not to hard-code the actual version number anywhere else but in `pyproject.toml`, otherwise it's just a matter of time until someone forgets to increment one or the other. But it should be fine to just put something like "dev" or "undetermined" here to indicate that the version number could not be loaded from the package info. Or does PyCharm not like that?